pkg/controller/core.oam.dev/v1alpha2/application/builder/build_test.go

```diff
@@ -0,0 +1,14 @@
+package builder
+
+import (
+	"testing"
+
+	"github.com/oam-dev/kubevela/pkg/controller/core.oam.dev/v1alpha2/application/parser"
+)
+
+func TestBuild(t *testing.T) {
+	_, _, err := Build("default", parser.TestExceptApp)
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
please add more test cases for builder. For server-side code, we have almost 65% unit test coverage, we should imporve our code quality and make it production ready ASAP.